centrifuge/src/main/java/io/github/centrifugal/centrifuge/Client.java

```diff
@@ -614,9 +625,12 @@ public void onDone(ReplyError error, Protocol.RefreshResult result) {
     }
 
     private void sendConnect() {
-        Protocol.ConnectRequest req = Protocol.ConnectRequest.newBuilder()
-                .setToken(this.token)
-                .build();
+        Protocol.ConnectRequest.Builder build=Protocol.ConnectRequest.newBuilder()
```

Comment:
I suppose sth wrong with formatting here and below? I mean no spaces between tokens.

Reply:
I just mean that you should use the same code formatting with spaces between `=` for example. And in other places.